test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -63,9 +63,11 @@ var timeouts = map[*regexp.Regexp]time.Duration{
 	regexp.MustCompile("pkg/network/usm$"):            30 * time.Minute,
 }
 
+// Note: security tests don't have the full path to the test, so increase the timeout as a temporary solution
+
 func getTimeout(pkg string) time.Duration {
 	matchSize := 0
-	to := 10 * time.Minute
+	to := 30 * time.Minute
```

Comment:
Why are we changing the default timeout for all tests?

Reply:
Related to the [comment above](https://github.com/DataDog/datadog-agent/pull/22336#discussion_r1466786248), because I cannot match for security agent tests I had to increase the default timeout.